docs/source/documentation.rst

```diff
@@ -433,6 +433,16 @@ If the segmentation with AxonDeepSeg fails or does not give optimal results, you
 * Manually correct the axon mask (as explained in Option 1).
 * Use the `AxonSeg <https://github.com/neuropoly/axonseg>`_ software to segment the myelin from the axon mask. In order to do this, install AxonSeg, and then follow the instructions in part 5 of the `as_tutorial guideline <https://github.com/neuropoly/axonseg/blob/master/as_tutorial.m>`_.
 
+**Installing FSLeyes**
+ 
+* Install the FSLeyes_integration branch of AxonDeepSeg inside a conda environment
+* Install Wxpython using conda install -c anaconda wxpython
+* Install FSLeyes using pip install fsleyes
+* Launch FSLeyes using the command fsleyes
+* Go to file > load plugin > select ads_plugin.py > yes
```

Comment:
At the moment, I get this error when trying to load the plugin:

<img width="438" alt="Capture d’écran 2019-04-03 à 13 21 57" src="https://user-images.githubusercontent.com/1421029/55493486-9246ed80-5613-11e9-8088-5f0f5adf682c.png">

I am on commit 8cb9387d146

Reply:
<img width="584" alt="Capture d’écran 2019-04-03 à 13 24 15" src="https://user-images.githubusercontent.com/1421029/55493605-d508c580-5613-11e9-8be9-0e50720960e7.png">

Reply:
Resolved in 14e3523